client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -63,6 +73,7 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 							autoFocusBlogname
 							blogname={ this.state.blogname }
 							blogdescription={ this.state.blogdescription }
+							disabled={ isRequestingSettings }
```

Comment:
I'd also disable the button in that case to make all form components behave consistently.

Also we'd probably want to prevent from doing anything in `handleSubmit` if we're requesting settings.

Reply:
8e235dd

Reply:
...and e22245b

Reply:
Awesome, thank you Bernie 👍